R/LearnerTorch.R

```diff
@@ -29,6 +29,21 @@
 #' To do so, you just need to include `epochs = to_tune(upper = <upper>, internal = TRUE)` in the search space,
 #' where `<upper>` is the maximally allowed number of epochs, and configure the early stopping.
 #'
+#' @section Network Head and Target Encoding:
+#' Torch learners are expected to have the following output:
+#' * binary classification: `c(batch_size, 1)`, representing the logits for the positive class.
+#' * multiclass classification: `c(batch_size, n_classes)`, representing the logits for all classes.
+#' * regression: `c(batch_size, 1)` representing the response prediction.
+#'
+#' Furthermore, the target encoding is expected to be as follows:
+#' * regression: The `numeric` target variable of a [`TaskRegr`][mlr3::TaskRegr] is encoded as a
+#'   [`torch_float`][torch::torch_float] with shape `c(batch_size, 1)`.
+#' * binary classification: The `factor` target variable of a [`TaskClassif`][mlr3::TaskClassif] is encoded as a
+#'   [`torch_float`][torch::torch_float] with shape `(batch_size, 1)` where the positive class is `1` and the negative
+#'   class is `0`.
```

Comment:
ok, but what is the positive and negative class at the R-level?
In R it is a factor with two levels. Is the first factor level always considered the positive class?

Reply:
`mlr3` binary classification tasks have a field `$positive`. Will add that to the docs!

Reply:
And yes, it is also ensured that the positive class is the first level.

Reply:
thanks